exampleSite/content/dev/resource-fallthrough/fragment/footer/index.md

```diff
@@ -1,14 +1,12 @@
 +++
 fragment = "footer"
-#disabled = true
 date = "2016-09-07"
 weight = 1200
-#background = ""
 
 [logo]
-  title = "Logo Title"
+  title = "logo"
   image = "resource_logo.svg"
-  text = "Logo Subtext"
+  text = "alt text"
```

Comment:
Haven't noticed this before. Isn't it better for this to be `alt`?

Reply:
I think we use text in other fragments as well. Such as logo fragment etc.